src/bin/core.js

```diff
@@ -14,16 +14,16 @@ async function build(passedPath) {
     const harmonic = new Harmonic(sitePath, { quiet: false });
 
     harmonic.start(); // useless, remove?
-    harmonic.clean();
-    harmonic.createPublicFolder();
+    await harmonic.clean();
+    await harmonic.createPublicFolder();
```

Comment:
We can probably remove the `createPublicFolder` method altogether, basically all the other methods already call `mkdirp`.
Need to test to make sure.